.github/workflows/msbuild.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       run: dotnet tool restore
     - name: Check code style
       working-directory: dafny/Source
-      run: dotnet tool run dotnet-format -w --check Dafny.sln
+      run: dotnet tool run dotnet-format -w --check Dafny.sln --exclude Dafny/Scanner.cs --exclude Dafny/Parser.cs
```

Comment:
Did this cause previous builds to fail? How come it didn't cause all previous builds to fail?